hypergan/gans/standard_gan.py

```diff
@@ -67,8 +68,16 @@ def forward_pass(self):
         self.x = self.inputs.next()
         g = self.generator(self.latent.next())
         self.g = g
-        d_real = self.forward_discriminator([self.x])
-        d_fake = self.forward_discriminator([g])
+        if self.config.diff_augment:
```

Comment:
Can we instead have forward_discriminator call a chain of `train_hook.augment_discriminator_input` and move this to a train hook?  This will allow us to have it work across multiple GANs and also chain DifferentialAugmentTrainHooks together.